src/docs/environment/index.mdx

```diff
@@ -151,12 +147,33 @@ for instructions on how to set up Bash.
 ```shell {filename: ~/.zprofile} {tabTitle: Zsh}
 
 # It is assumed that pyenv is installed via Brew, so this is all we need to do.
-eval "$(pyenv init --path)"
-```
+
+eval "\$(pyenv init --path)"
+
+````
 
 ```shell {filename: ~/.config/fish/config.fish} {tabTitle: Fish}
 # pyenv init
 status is-login; and pyenv init --path | source
+````
+
+### (Optional) Setup `fish` shell
+
+If you use [fish](https://fishshell.com/) you need to manually set some of the environment variables. This only needs to be done once.
+
+```shell {tabTitle: Fish}
+set -Ux PYENV_ROOT $HOME/.pyenv
+# fish >=3.2.0
+fish_add_path $PYENV_ROOT/bin
+# fish <3.2.0
+set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+```
+
+Once that's done, your shell needs to be reloaded. You can either reload it in-place, or close your terminal and start it
+again and cd into sentry. To reload it, run:
+
+```shell
+exec "$SHELL"
 ```
```

Comment:
ideally we should link to pyenv docs here rather than trying to reproduce their docs (probably with drift and/or poorly) -- alternatively I'd be fine just not supporting `fish` since it has a bunch of other non-standard quirks and presumably if you choose fish you'd know how to work around this already

Reply:
I agree. Lets remove fish